clients/cli/cmd/internal/print/print.go

```diff
@@ -52,14 +52,24 @@ func NonBatchPrintf(msg string, args ...interface{}) {
 
 func Success(msg string, args ...interface{}) {
 	if shared.BatchMode {
-		jsonStuct := &InfoMessage{Message: fmt.Sprintf(msg, args...)}
-		encodedJson, _ := json.Marshal(jsonStuct)
+		jsonStruct := &InfoMessage{Message: fmt.Sprintf(msg, args...)}
+		encodedJson, _ := json.Marshal(jsonStruct)
 		fmt.Println(string(encodedJson))
 	} else {
 		WithColor(ct.Green, msg, args...)
 	}
 }
 
+func Warn(msg string, args ...interface{}) {
+	if shared.BatchMode {
+		jsonStruct := &InfoMessage{Message: fmt.Sprintf(msg, args...)}
+		encodedJson, _ := json.Marshal(jsonStruct)
+		fmt.Println(string(encodedJson))
+	} else {
+		WithColor(ct.Yellow, msg, args...)
+	}
```

Comment:
I guess it makes sense to extract this now into a function like `BatchSensitivePrint(string, color, args)` or something?

Reply:
Was also thinking whether it should be extract as the building the JSON seems quite similar. But for example the error message looks differently from the other messages. That makes unifying more difficult :thinking: